tests/k6/tests/scripts/generate_tokens.sh

```
@@ -0,0 +1,94 @@
+    fi
+    echo "org,orgno,scopes,resource,token" > $serviceowner_tokenfile
+    while IFS=, read -r org orgno scopes resource
+    do
+        url="https://altinn-testtools-token-generator.azurewebsites.net/api/GetEnterpriseToken?org=$org&env=$env&scopes=$scopes&orgno=$orgno&ttl=3600"
+        token=$(curl -s -f $url -u "$tokengenuser:$tokengenpasswd" )
+        if [ $? -ne 0 ]; then
+            echo "Error: Failed to generate enterprise token for: $env, $org, $orgno, $scopes "
+            continue
+        fi
+        echo "$org,$orgno,$scopes,$resource,$token" >> $serviceowner_tokenfile
+        status=$?
```

Shellcheck

[warning] This $? refers to echo/printf, not a previous command. Assign to variable to avoid it being overwritten.

(SC2320)